### PR TITLE
yamllint formatting cleanup for ocp-workload-pam7-cc-dispute-dmn-pmml

### PR DESCRIPTION
**Pull request by ritesh97-rh**
##### SUMMARY
For the specified roles:
- Create a yamllint file to enforce formatting rules
- Make changes to existing code base to adhere to formatting requirements
- yamllint errors will be displayed during formatting checks in future development

##### ISSUE TYPE
- Code Cleanup Request

##### COMPONENT NAME
ocp-workload-pam7-cc-dispute-dmn-pmml

### CODE REVIEW

**Review by stencell — CHANGES_REQUESTED** [review]
This one has changes to commands that remove escaped chars. It needs to be tested before merging.

**Comment by ritesh97-rh** [comment]
Would talk to Jonathan about testing. I'm only on this project for a short period.

File: ansible/roles/ocp-workload-pam7-cc-dispute-dmn-pmml/tasks/prometheus_workload.yml [hunk 1]
```diff
@@ -31,4 +37,9 @@
 
 # Patch the OAuth proxy to allow Bearer token authentication
 - name: Patch OAuth proxy
-  shell: "oc patch statefulset/prom --type='json' -p=\"[{'op': 'add', 'path': '/spec/template/spec/containers/0/args/-', 'value': '-openshift-delegate-urls={\\\"/\\\":{\\\"resource\\\":\\\"pods\\\",\\\"namespace\\\":\\\"{{ OCP_PROJECT }}\\\",\\\"name\\\":\\\"prom\\\",\\\"verb\\\":\\\"get\\\"}}'}]\" -n {{ OCP_PROJECT }}"
+  shell: >-
+    oc patch statefulset/prom --type='json'
+    -p="[{'op': 'add', 'path': '/spec/template/spec/containers/0/args/-',
+    'value': '-openshift-delegate-urls={\"/\":{\"resource\":\"pods\",\"namespace\":\"{{
+      OCP_PROJECT }}\",\"name\":\"prom\",\"verb\":\"get\"}}'}]"
+      -n {{ OCP_PROJECT }}
```
Comment (jkupferer) [inline ansible/roles/ocp-workload-pam7-cc-dispute-dmn-pmml/tasks/prometheus_workload.yml:45]: This line breaking is really hard to read.

File: ansible/roles/ocp-workload-pam7-cc-dispute-dmn-pmml/tasks/rhpam_openshift_dmn_pmml_workload.yml [hunk 2]
```diff
@@ -1,60 +1,104 @@
...
+    --image-stream="openshift/rhpam-kieserver-rhel8:7.5.0" --binary
+    --name rhpam-kieserver-rhel8-dmn-pmml
+    -n {{ OCP_PROJECT }}
 
 - name: "Configure Business Central ImageStream"
-  shell: "oc patch dc/{{ pam_app_name }}-rhpamcentr --type='json' -p '[{\"op\": \"replace\", \"path\": \"/spec/triggers/0/imageChangeParams/from/name\", \"value\": \"rhpam-businesscentral-rhel8-dmn-pmml:latest\"}]' -n {{ OCP_PROJECT }}"
+  shell: >-
+    oc patch dc/{{ pam_app_name }}-rhpamcentr --type='json'
+    -p '[{"op": "replace", "path": "/spec/triggers/0/imageChangeParams/from/name",
+    "value": "rhpam-businesscentral-rhel8-dmn-pmml:latest"}]'
```
Comment (jkupferer) [inline ansible/roles/ocp-workload-pam7-cc-dispute-dmn-pmml/tasks/rhpam_openshift_dmn_pmml_workload.yml:28]: Line break in the middle of an argument?

**Review by jkupferer — CHANGES_REQUESTED** [review]

**Pull request closed without merging**